queries.sql

```diff
@@ -9,3 +9,67 @@ SELECT name FROM animals WHERE weight_kg < 10.5;
 SELECT name FROM animals WHERE neutered = TRUE;
 SELECT name FROM animals WHERE name != 'Gabumon';
 SELECT name FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```

Comment:
- [ ] Please make sure to write [the following query](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction%20update%20the%20animals%20table%20by%20setting%20the%20species%20column%20to%20unspecified.%20Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.) which updates the species in the "animals" table.